src/DbCache.php

```diff
@@ -35,6 +37,8 @@
  */
 final class DbCache implements CacheInterface
 {
+    use LoggerAwareTrait;
```

Comment:
PSR logger should be added to composer.json suggest section.

Reply:
done.